requirements.txt

```diff
@@ -1,6 +1,6 @@
 django>=3.0,<3.2
 swapper~=1.1.0
-openwisp-utils[rest]~=0.7
+openwisp-utils[rest]~=0.7.3
```

Comment:
please let's change openwisp-utils in requirements.txt and requirement-tests.txt exactly as we're doing in this PR:
https://github.com/openwisp/openwisp-radius/pull/203